src/components/YourCompany/CorrectBusinessType.vue

```diff
@@ -0,0 +1,352 @@
+                  id="btn-more-actions-edit"
+                  @click="isEditingType = true; dropdown = false"
+                >
+                  <v-list-item-subtitle>
+                    <v-icon small color="primary">mdi-pencil</v-icon>
+                    <span class="drop-down-action ml-1">Change</span>
+                  </v-list-item-subtitle>
+                </v-list-item>
+              </v-list>
+            </v-menu>
+          </span>
+        </div>
+      </v-flex>
+    </v-layout>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Emit, Mixins, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+
+// Components
+import { ContactInfo } from '@/components/common'
+
+// Mixins
+import { CommonMixin } from '@/mixins'
+
+// Enums and Interfaces
+import { EntityTypes } from '@/enums'
+import { ActionBindingIF, BusinessSnapshotIF } from '@/interfaces'
+
+@Component({
+  components: {
+    ContactInfo
+  }
+})
+export default class CorrectBusinessType extends Mixins(CommonMixin) {
+  @Getter getEntityType!: EntityTypes
+  @Getter isConflictingLegalType: boolean
+  @Getter getOriginalSnapshot: BusinessSnapshotIF
+
+  @Action setEntityType!: ActionBindingIF
+
+  readonly EntityTypes = EntityTypes
+
+  private selectedEntityType: EntityTypes = null
+  private confirmArticles: boolean = false
+  private helpToggle: boolean = false
+  private isEditingType = false
+
+  /** V-model for dropdown menu. */
+  private dropdown: boolean = null
+
+  /** Define the entity type locally once the value has been populated in the store. */
+  @Watch('getEntityType')
+  private initializeEntityType () {
+    this.selectedEntityType = this.getEntityType
+  }
+
+  /** Entity Options. */
+  private entityTypeOptions = [
+    {
+      value: 'CR',
+      SHORT_DESC: 'BC Limited Company',
+      text: 'BC Limited Company'
+    },
+    {
+      value: 'BEN',
+      SHORT_DESC: 'BC Benefit Company',
+      text: 'BC Benefit Company'
+    }
+  ]
+
+  /** Verify New Business type. */
+  private get isNewBusinessType (): boolean {
+    return this.getEntityType && this.getEntityType !== this.getOriginalSnapshot[0]?.business?.legalType
+  }
+
+  /** Reset company type values to original. */
+  private resetType () {
+    this.setEntityType(this.getOriginalSnapshot[0]?.business?.legalType)
+    this.emitHaveChanges(false)
+    this.isEditingType = false
+    this.confirmArticles = false
+  }
+
+  /** Reset company type values to original. */
+  private submitTypeChange () {
+    this.setEntityType(this.selectedEntityType)
+    this.emitHaveChanges(true)
+    this.isEditingType = false
+  }
+
+  @Emit('haveChanges')
+  private emitHaveChanges (haveChanges: boolean): void {}
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+ol {
+  counter-reset: item;
+}
+
+ol > li {
+  counter-increment: item;
+}
+
+ol ol > li {
+  display: block;
+}
+
+ol ol > li:before {
```

Comment:
Would it be clearer/more concise to nest this under the selectors above?

Reply:
For sure, can do! I think everything is scoped on the component but that wouldn't hurt as well

Reply:
My comment is not about the scope but about the apparent duplication of selectors, which can be make clear by nesting.

Reply:
Ahh i see, ok will refactor this 👍

Reply:
fixed in next commit